dom0/sd-dom0-crontab.sls

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+##
+# Update /etc/crontab with any cron jobs we need to run regularly
+##
+
+# Identify the GUI user by group membership
+{% set gui_user = salt['cmd.shell']('groupmems -l -g qubes') %}
+
+# Add an hourly job, run as the GUI user, to display a warning if the
+# SecureDrop preflight updater has not run for longer than a defined
+# warning threshold.
+dom0-crontab-update-notify:
+  file.blockreplace:
+    - name: /etc/crontab
```

Comment:
`/cron.hourly` will run as root, so modifying `/etc/crontab` appears to be the cleanest way to run as the gui_user, which is necessary in order to run with the same permissions as the preflight updater.